src/nss-systemd/nss-systemd.c

```diff
@@ -6,8 +6,8 @@
 #include "env-util.h"
 #include "errno-util.h"
 #include "fd-util.h"
-#include "group-record-nss.h"
 #include "macro.h"
+#include "nss-record.h"
```

Comment:
as earlier, i'd probably call this nss-user-record.h still, i.e. let the "group" part fall under the table, but let's keep the "user" in there, it's relevant I think, because we have NSS for other records, too, such as hostnames...